samples/apps/txregulator/tests/txregulatorclient.py

```diff
@@ -99,20 +99,20 @@ def run(args):
         # Check permissions are enforced
         with primary.user_client(user_id=regulator.name) as c:
             check(
-                c.rpc("REG_register", {}),
+                c.rpc("REG_register"),
                 error=check_status(http.HTTPStatus.FORBIDDEN),
             )
             check(
-                c.rpc("BK_register", {}), error=check_status(http.HTTPStatus.FORBIDDEN),
+                c.rpc("BK_register"), error=check_status(http.HTTPStatus.FORBIDDEN),
             )
 
         with primary.user_client(user_id=banks[0].name) as c:
             check(
-                c.rpc("REG_register", {}),
+                c.rpc("REG_register"),
```

Comment:
That probably wants to become “regulator/register” now.

Reply:
Agreed: https://github.com/microsoft/CCF/issues/968